docs/about/contributing.md

```diff
@@ -41,148 +41,158 @@ We use [semantic-release](https://www.npmjs.com/package/semantic-release), which
+* For a breaking change: `feat(scm): Support new scm plugin. BREAKING CHANGE: github no longer works`
 
 ## Feature Requests
 
-Make the case for a feature via an issue with a good title. The feature should be discussed and given a target inclusion milestone or closed.
+Make the case for a feature via an issue with a good title. The feature will then be discussed and either given a target inclusion milestone or closed.
 
 ## Where to Contribute
 
-Screwdriver has a modular architecture, and the various responsibilities are split up in separate repos.
+Screwdriver has a modular architecture, and the various responsibilities are split up into separate repos.
+
+Check out the [architecture diagram](arch-diagram) to see the overall workflow of Continuous Delivery using Screwdriver. The next few sections will help lay out where different code repositories fit.
 
-### [Screwdriver API][api-repo] [![Version][api-npm-image]][api-npm-url] [![Open Issues][api-issues-image]][api-issues-url]
+### [Screwdriver API][api-repo]
 The **[screwdriver][api-repo]** repo is the core of screwdriver, providing the API endpoints for everything that screwdriver does. The API is based on the *[hapijs framework](http://hapijs.com/)* and is implemented in node as a series of plugins.
 
+
+* **[Build bookends][build-bookend-repo]** allow a user to create setup and teardown steps for builds.
+* The API can also send notifications to users. [notifications-base][notifications-base-repo] is the base class for defining the behavior between screwdriver and notifications plugins, like [email notifications][notifications-email-repo].
+
 ### [Launcher][launcher-repo]
 
-The **[launcher][launcher-repo]** performs step execution and housekeeping internal to build containers. This is written in Go, and mounted into build containers as a binary.
+The **[launcher][launcher-repo]** performs step execution and housekeeping internal to build containers. This is written in Go and mounted into build containers as a binary.
+
+* **[sd-step][sd-step-repo]**: ???
+* **[meta-cli][meta-cli-repo]**: A Go-based CLI for reading/writing information from the metadata
 
 ### Executors
 
 An executor is used to manage build containers for any given job. Several implementations of executors have been created. All are designed to follow a common interface. Executor implementations are written in node:
 
-* **[executor-base][executor-base-repo]**: Common interface [![Version][executor-base-npm-image]][executor-base-npm-url]
-* **[executor-docker][executor-docker-repo]**: Docker implementation [![Version][executor-docker-npm-image]][executor-docker-npm-url]
-* **[executor-j5s](https://github.com/screwdriver-cd/executor-j5s)**: Jenkins implementation [![Version][executor-j5s-npm-image]][executor-j5s-npm-url]
-* **[executor-k8s][executor-k8s-repo]**: Kubernetes implementation [![Version][executor-k8s-npm-image]][executor-k8s-npm-url]
+* **[executor-base][executor-base-repo]**: Common interface
```

Comment:
It's included below in line 87: "The [executor router][executor-router-repo] is a generic executor plugin that routes builds to a specified executor."

Do you want me to move it up?